lib/kredis/types/unique_list.rb

```diff
@@ -1,28 +1,28 @@
 # You'd normally call this a set, but Redis already has another data type for that
 class Kredis::Types::UniqueList < Kredis::Types::List
-  proxying :multi, :ltrim, :exists?
+  proxying :multi, :exists?
 
   attr_accessor :typed, :limit
 
   def prepend(elements)
     elements = Array(elements).uniq
     return if elements.empty?
 
-    multi do
-      remove elements
+    multi do |pipeline|
+      types_to_strings(elements, typed).each { |element| pipeline.lrem key, 0, element }
```

Comment:
It repeats `List#remove`'s code, but making `remove` accept a pipeline looked equally unideal.

Reply:
Man, this is a pretty nasty API regression. Need to take a look and see what else we can come up with.